cmd/flowserver/main.go

```diff
@@ -18,7 +18,7 @@ import (
 
 	"github.com/go-chi/chi/middleware"
 	"github.com/pressly/lg"
-	"github.com/sirupsen/logrus"
```

Comment:
seems for our limited logging, using the package level standard logger is easier than finding a way to pass a logger instance all the way down to events